ormconfig.js

```diff
@@ -12,23 +12,40 @@ if (process.env.NODE_ENV === 'test') {
   entities = 'src/entities/**/*.ts';
 }
 
-module.exports = {
-  type: 'postgres',
-  url: process.env.DATABASE_URL,
-  ssl: true,
-  extra: {
-    ssl: {
-      rejectUnauthorized: false,
+let prodSSLConfig = { ssl: true, extra: { ssl: { rejectUnauthorized: false } } };
+let localSSLConfig = { ssl: false, extra: {} };
+let sslConfig = process.env.NODE_ENV === 'development' ? localSSLConfig : prodSSLConfig;
+
+module.exports = [
+  {
```

Comment:
What is the difference between this and the "seed" object array element?

Reply:
this just specifies 2 separate connections, it's an abuse to use seeds as separate "migrations"